tensorflow/python/ops/array_ops.py

```diff
@@ -19,6 +19,7 @@
 import numpy as np
 
 from tensorflow.core.config import flags
+from tensorflow.python.autograph.lang import directives
```

Comment:
We need a dep on
```
//tensorflow/python/autograph/lang:directives
```
in the BUILD file for this.

Reply:
Ah good spot, added this in